pygeo/parameterization/DVGeo.py

```diff
@@ -16,7 +17,7 @@
 
 
 class DVGeometry:
-    """
+    r"""
```

Comment:
Why does the string need to be raw?
Is it related to how it is read by the docs generator?

Reply:
This is sort of a superficial change. By making it raw strings, the LaTeX math stuff no longer need to be escaped with double backslashes when rendered in Sphinx. Either are valid, but this is a bit easier to read in the source code.

Reply:
:+1: